extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventActionExecutionFieldsMixin.json

```diff
@@ -0,0 +1,92 @@
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journeyStepEventActionExecutionFields",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey Orchestration Step Event Action Execution Fields",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journeyStepEvent",
+    "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
+  "description": "This Mixin will have Journey Action Metadata Fields",
+  "definitions": {
+    "journeyStepEventActionExecutionFields": {
+      "properties": {
+        "https://ns.adobe.com/experience/journeyOrchestration/stepEvents": {
+          "title": "Journey Orchestration for creating user journey.",
+          "type": "object",
+          "properties": {
+            "journey:actionId":{
+              "title":"actionId",
+              "description":"Id of the action that is being executed",
+              "type":"string"
+            },
+            "journey:actionName":{
+              "title":"actionName",
+              "description":"Name of the action If no name has been set, the stepName will be taken",
+              "type":"string"
+            },
+            "journey:actionType":{
+              "title":"actionType",
+              "description":"Type of the action",
+              "type":"string"
+            },
+            "journey:actionParameterized":{
+              "title":"actionParameterized",
+              "description":"Indicates if the action is parameterized or not",
+              "type":"boolean"
+            },
+            "journey:actionExecTime":{
+              "title":"actionExecTime",
+              "description":"The amount of time (in millis) taken to execute a current action",
+              "type":"integer"
+            },
+            "journey:actionExecError":{
+              "title":"actionExecError",
+              "description":"Type of error that happens when the action is called.",
+              "type":"string"
+            },
+            "journey:actionExecErrorCode":{
+              "title":"actionExecErrorCode",
+              "description":"Code for action execution error. Present if the error has a code, such as an HTTP one",
+              "type":"string"
+            },
+            "journey:actionExecOrigError":{
+              "title":"actionExecOrigError",
+              "description":"",
+              "type":"string"
+            },
+            "journey:actionExecOrigErrorCode":{
+              "title":"actionExecOrigErrorCode",
+              "description":"error code of the actionExecOrigError",
+              "type":"string"
```

Comment:
Number?

Reply:
we are using the string as the Errors are not HTTP but HTTP, Capping Errors from service, etc.